src/modules/categories/components/Categories.js

```diff
@@ -159,7 +159,7 @@ class Categories extends React.Component<PropsType> {
       return <SpaceBetween>
         <View>
           <Tiles tiles={this.getTileModels(children)}
-                 title={CityModel.findCityName(cities, category.title)}
+                 title={null}
```

Comment:
I think it would make more sense to use undefined here and just remove the `tile=null`